src/components/Repos/Repo.js

```diff
@@ -1,45 +1,72 @@
 import { useState, useEffect } from 'react'
-import { useParams } from "react-router-dom"
+import { useParams, Link } from "react-router-dom"
 
 function Repo () {
-  const [repo, setRepo] = useState({})
+  const [repo, setRepo] = useState({notes: []})
   const [notFound, setNotFound] = useState(false)
 
-  const params = useParams()
-  console.log(params)
+  const {username, reponame} = useParams()
 
   useEffect(() => {
-    //  https://api.github.com/repos/OWNER/REPO
-    fetch(`https://api.github.com/repos/${params.username}/${params.reponame}`)
-      .then(res => res.json()) // read the response format which is stored in JSON
-      .then(data => {
-      if (data.message === 'Not Found') {
+    const getRepoAndNotes = async () => {
+      const ghRes = await fetch(`https://api.github.com/repos/${username}/${reponame}`)
+      const ghData = await ghRes.json()
+      if (ghData.message === 'Not Found') {
         setNotFound(true)
       } else {
-        console.log(data)
         setNotFound(false)
-        setRepo(data)
+        const notesRes = await fetch(`http://localhost:4000/notes?repo=${ghData.name}`)
+        const notes = await notesRes.json()
+        setRepo({...ghData, notes})
       }
-    })
-  }, [])
+    }
+
+    getRepoAndNotes()
+  }, [username, reponame])
 
+  const sortedNotes = () => {
+    return repo.notes.sort((a, b) => Date.parse(b.timestamp) - Date.parse(a.timestamp))
```

Comment:
Could you explain the use of a and b parameters here please?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort